diff --git a/node_modules/react-native-css-interop/.cache/android.js b/node_modules/react-native-css-interop/.cache/android.js
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/react-native-css-interop/.cache/ios.js b/node_modules/react-native-css-interop/.cache/ios.js
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/react-native-css-interop/.cache/macos.js b/node_modules/react-native-css-interop/.cache/macos.js
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/react-native-css-interop/.cache/native.js b/node_modules/react-native-css-interop/.cache/native.js
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/react-native-css-interop/.cache/windows.js b/node_modules/react-native-css-interop/.cache/windows.js
new file mode 100644
index 0000000..e69de29
diff --git a/node_modules/react-native-css-interop/dist/metro/index.js b/node_modules/react-native-css-interop/dist/metro/index.js
index c91ba6a..fbf3e67 100644
--- a/node_modules/react-native-css-interop/dist/metro/index.js
+++ b/node_modules/react-native-css-interop/dist/metro/index.js
@@ -176,7 +176,7 @@ async function startCSSProcessor(filePath, platform, isDev, { input, getCSSForPl
                 ? css
                 : getNativeJS((0, css_to_rn_1.cssToReactNativeRuntime)(css, options), debug)));
             debug(`virtualStyles.emit ${platform}`);
-            haste.emit("change", {
+            haste.emit("change", { changes: { addedFiles: [], modifiedFiles: [], removedFiles: [] }, rootDir: "", changes: { addedFiles: [], modifiedFiles: [], removedFiles: [] }, rootDir: "",
                 eventsQueue: [
                     {
                         filePath,
diff --git a/node_modules/react-native-css-interop/src/metro/index.ts b/node_modules/react-native-css-interop/src/metro/index.ts
index fa58c7b..85c9123 100644
--- a/node_modules/react-native-css-interop/src/metro/index.ts
+++ b/node_modules/react-native-css-interop/src/metro/index.ts
@@ -341,7 +341,7 @@ async function startCSSProcessor(
         );
 
         debug(`virtualStyles.emit ${platform}`);
-        haste.emit("change", {
+        haste.emit("change", { changes: { addedFiles: [], modifiedFiles: [], removedFiles: [] }, rootDir: "", changes: { addedFiles: [], modifiedFiles: [], removedFiles: [] }, rootDir: "",
           eventsQueue: [
             {
               filePath,
